start_cerberus.py

```diff
@@ -138,8 +139,18 @@ def main(cfg):
             logging.info("Detailed inspection of failed components has been enabled")
             inspect.delete_inspect_directory()
 
-        # get list of all master nodes to verify scheduling
-        master_nodes = kubecli.list_nodes("node-role.kubernetes.io/master")
+        # get list of all master nodes with provided labels in the config
+        master_label = watch_master_schedulable["label"]
+        master_nodes = []
+        if watch_master_schedulable["enabled"]:
+            nodes = kubecli.list_nodes(master_label)
+            if len(nodes) == 0:
+                logging.error("No master node found for the label %s \
```

Comment:
This spacing gets outputted very strangely. Not sure if we can rework this to be under the line length requirements and output nicely. 

<img width="1396" alt="Screen Shot 2021-04-21 at 11 53 14 AM" src="https://user-images.githubusercontent.com/64206430/115583795-3867fd80-a298-11eb-8f7b-7b90b26eb9a9.png">

Reply:
@amitsagtani97 are you able to take a look into this weird spacing

Reply:
fixed it